src/main/drivers/timer.h

```diff
@@ -111,6 +111,8 @@ typedef enum {
     TIM_USE_MC_CHNFW        = (1 << 4),     // Deprecated and not used after removal of CHANNEL_FORWARDING feature
     TIM_USE_FW_MOTOR        = (1 << 5),
     TIM_USE_FW_SERVO        = (1 << 6),
+    TIM_USE_HC_MOTOR        = (1 << 7),     // woga65: Helicopter motor output
```

Comment:
With the new pwm assignment I am considering merging this into a single MOTOR / SERVO item. It could reduce the number o changes needed to targets. Otherwise you could use the FW mapping, as it more closely matches the heli use case (few motors, many servos)

Reply:
> ... I am considering merging this into a single MOTOR / SERVO item...
This would be my favourite choice. I am excited about the new pwm assignment and would like to adapt my code to it as soon as I can get my hands on it. 
For now I could `TIM_USE_HC_MOTOR = TIM_USE_FW_MOTOR` and `TIM_USE_HC_SERVO = TIM_USE_FW_SERVO`. This would keep things separated from the perspective of someone who needs to to read the code while reducing the changes to single target files.
Additionally I could define `USE_VARIABLE_PITCH` globally for all targets with sufficient flash memory space and introduce a new field in MSP_BOARD_INFO to tell the configurator whether a target supports helicopters or not.

Reply:
> > ... I am considering merging this into a single MOTOR / SERVO item...
> 
> This would be my favourite choice. I am excited about the new pwm assignment and would like to adapt my code to it as soon as I can get my hands on it. For now I could `TIM_USE_HC_MOTOR = TIM_USE_FW_MOTOR` and `TIM_USE_HC_SERVO = TIM_USE_FW_SERVO`. This would keep things separated from the perspective of someone who needs to to read the code while reducing the changes to single target files. Additionally I could define `USE_VARIABLE_PITCH` globally for all targets with sufficient flash memory space and introduce a new field in MSP_BOARD_INFO to tell the configurator whether a target supports helicopters or not.

The code has been merged to master.
Defining TIM_USE_HC_ to match TIM_USE_FW_ is a good solution to keep things gong and simplify for now.

Reply:
> > > ... I am considering merging this into a single MOTOR / SERVO item...
> > 
> > 
> > This would be my favourite choice. I am excited about the new pwm assignment and would like to adapt my code to it as soon as I can get my hands on it. For now I could `TIM_USE_HC_MOTOR = TIM_USE_FW_MOTOR` and `TIM_USE_HC_SERVO = TIM_USE_FW_SERVO`. This would keep things separated from the perspective of someone who needs to to read the code while reducing the changes to single target files. Additionally I could define `USE_VARIABLE_PITCH` globally for all targets with sufficient flash memory space and introduce a new field in MSP_BOARD_INFO to tell the configurator whether a target supports helicopters or not.
> 
> The code has been merged to master. 

Nice! Can it already be configured in the Configurator's GUI, or just CLI for now?

> Defining TIM_USE_HC_ to match TIM_USE_FW_ is a good solution to keep things gong and simplify for now.

I will do that way.